railties/lib/rails/generators/test_case.rb

```diff
@@ -25,7 +25,7 @@ module Generators
     #     destination File.expand_path("../tmp", File.dirname(__FILE__))
     #     setup :prepare_destination
     #   end
-    class TestCase < ActiveSupport::TestCase
+    class TestCase < ApplicationTestCase
```

Comment:
But `ApplicationTestCase` make not sense since the application may have view tests, controller tests, integrations tests, model tests. If we add new test case superclasses we would need `ApplicationViewTestCase`, `ApplicationModelTestCase`. I think this is a problem to the application solve. If they are doing things to `ActiveSupport::TestCase` just because it is shared across all they test cases probably they want to add only in the places the thing is needed.